src/Snt/Capi/Repository/Article/FindByIdsParameters.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Snt\Capi\Repository\Article;
+
+final class FindByIdsParameters
+{
+    /**
+     * @var string
+     */
+    private $publicationId;
+
+    /**
+     * @var int[]
+     */
+    private $articleIds;
+
+    /**
+     * @param string $publicationId
+     * @param int[] $articleIds
+     */
+    public function __construct($publicationId, array $articleIds)
+    {
+        $this->publicationId = $publicationId;
+        $this->articleIds = $articleIds;
+    }
```

Comment:
minor improvement - what do you think about introducing named constructor, i.e. `createForPublicationAndArticleIds` or even `createFrom` with array argument?